index.html

```diff
@@ -8,7 +8,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>LinuxChix India</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```

Comment:
according to my opinion there is no need to allow the user to zoom in/out because that just make the website look vulnerable on mobile phones but if you want me to undo this change then I can do that since its your website.